themes/fast-track-theme/lms/static/sass/lms-course.scss

```diff
@@ -138,7 +138,7 @@ body.view-in-course {
       position: absolute;
       left: -20px;
       right: -20px;
-      top: 45%;
+      top: 350px;
```

Comment:
prev and next buttons are should be at the same position no matter how much course content is displayed.